<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>576</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>14</number>
   </property>
   <property name="topMargin">
    <number>14</number>
   </property>
   <property name="rightMargin">
    <number>14</number>
   </property>
   <property name="bottomMargin">
    <number>14</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">
QFrame{background:#171a21;border-radius:16px;}
QLabel{color:white;}
QComboBox, QSlider {color:white;}
QPushButton{height:50px;border-radius:12px;color:white;background:#222634;font-size:15px;padding:8px;}
QPushButton:pressed{background:#2d3245;}
      </string>
     </property>
     <layout class="QVBoxLayout" name="layoutInner">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="styleSheet">
         <string notr="true">QLabel{font-size:22px;font-weight:800;}</string>
        </property>
        <property name="text">
         <string>Paramètres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblBluetoothTitle">
        <property name="styleSheet">
         <string notr="true">QLabel{font-size:18px;font-weight:bold;margin-top:10px;}</string>
        </property>
        <property name="text">
         <string>Gestion Bluetooth</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="rowBluetoothButtons">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnVisible">
          <property name="text">
           <string>Rendre Visible (Appairage)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnForget">
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color:#852222;}</string>
          </property>
          <property name="text">
           <string>Oublier l'appareil</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblPaired">
        <property name="styleSheet">
         <string>color:#8892a0; font-size:14px;</string>
        </property>
        <property name="text">
         <string>Appareils enregistrés :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listDevices">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget { background: #2a2f3a; color: white; border-radius: 8px; font-size: 16px; padding: 5px; }
QListWidget::item { height: 40px; }
QListWidget::item:selected { background: #2a75ff; }</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacerV">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
